app/components/participant-creator/template.hbs

```diff
@@ -45,7 +45,7 @@
                                     <label class="text-muted">
                                         Use as password?
                                     </label>
-                                    <input name="useAsPassword" onchange={{action (action 'useAsPassword' item.key) value="target.checked"}} type="radio" />
+                                    <input name="useAsPassword" checked={{if useAsPassword true}} onchange={{action (action 'useAsPassword' item.key) value="target.checked"}} type="radio" />
```

Comment:
It is slightly confusing to have both a variable and an action by the same name in this controller. Can you please rename one to help differentiate?